app/routes/tasks/task.py

```diff
@@ -180,24 +198,14 @@ async def _check_completed(asset_id: UUID):
     )
 
     if all_finished:
+        # Set the asset to status saved
         asset_row: ORMAsset = await assets.update_asset(
             asset_id,
             status=AssetStatus.saved,
             change_log=[status_change_log.dict(by_alias=True)],
         )
-        # For database tables, fetch list of fields and their types from PostgreSQL
-        # and add them to metadata object
-        # Check if creation options specify to register a dynamic vector tile cache asset
-        if is_database_asset(asset_row.asset_type):
-            asset_row = await _update_asset_field_metadata(
-                asset_row.dataset, asset_row.version, asset_id,
-            )
-
-            await _register_dynamic_vector_tile_cache(
-                asset_row.dataset, asset_row.version, asset_row.metadata
-            )
 
-        # If default asset, make sure, version is also set to saved
+        # If default asset, make sure version is also set to saved
```

Comment:
Set version to default should run after `post_completion_task` in case that step fails

Reply:
Done.